src/main/java/com/hillel/items_exchange/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.hillel.items_exchange.mapper;
+
+import com.hillel.items_exchange.dto.CategoryVo;
+import com.hillel.items_exchange.dto.SubcategoryVo;
+import com.hillel.items_exchange.model.Category;
+import com.hillel.items_exchange.model.Subcategory;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@RequiredArgsConstructor
+public class CategoryMapper {
+    private final SubcategoryMapper subcategoryMapper;
+
+    public Category categoryVoToNewCategory(CategoryVo categoryVo) {
+        Category newCategory = new Category();
```

Comment:
You could refactor current logic - to create additional method with 3 parameters and create and fill Category into it. You do almost the same 3 times into current class.